politeiawww/client/ticketvote.go

```diff
@@ -203,23 +203,54 @@ func TicketVoteTimestampsVerify(tr tkv1.TimestampsReply) error {
 	// Verify authorization timestamps
 	for k, v := range tr.Auths {
 		err := TicketVoteTimestampVerify(v)
-		if err != nil {
+		switch {
+		case err == nil:
+			// Timestamp is valid and digest was included in a DCR tx.
+			continue
+
+		case err == backend.ErrNotTimestamped:
+			// Timestamp is valid but digest was not included in a DCR tx,
+			// continue.
+			continue
+
+		default:
+			// Unknown error
 			return fmt.Errorf("verify authorization %v timestamp: %v", k, err)
 		}
```

Comment:
Thought to add here a 
`log.Debugf("Verified %v vote timestamps; included in a DCR tx: %v, not included: %v")`\

We will need to access the `log` here in order to do that, @lukebp what do you think ? worths the effort ?